sentry.client.config.ts

```diff
@@ -4,27 +4,30 @@
 
 import * as Sentry from '@sentry/nextjs'
 
-Sentry.init({
-    dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
-    tracesSampleRate: 1,
-    enabled: true,
-    // Setting this option to true will print useful information to the console while you're setting up Sentry.
-    debug: false,
-    replaysOnErrorSampleRate: 1.0,
+if (process.env.NODE_ENV !== 'development') {
```

Comment:
thought: this never worked properly. still uploads to sentry everytime i build locally, and it takes forever. Kills me